microshift_backup_and_restore/microshift-backup-and-restore.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: ASSEMBLY
+[id="microshift-backup-and-restore"]
+= About backing up and restoring {product-title} data
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-backup-and-restore
+
+toc::[]
+
+You can manually backup and restore the {product-title} database. If you are using an RPM-OSTree system, backup and restore are automated. If you are using a non-OSTree system, then you can manually back up and restore {product-title} data. The {product-title} service and Greenboot health checks must be stopped prior to any backups.
```

Comment:
I don't think we used term "RPM-OSTree" anywhere. We referred to the non-ostree system as "regular RPM system" or "RPM system".

However I see that [R4E docs](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/composing_installing_and_managing_rhel_for_edge_images/introducing-rhel-for-edge-images_composing-installing-managing-rhel-for-edge-images) use "`rpm-ostree` image".
So perhaps that's the way it should be in the docs? Just my 2 cents

Reply:
Funny enough, I had settled on "RPM-OSTree" for our docs, but I just looked it up in OCP Supplementary Style Guide, and I see: "RPM, rpm-ostree, OSTree," so the usage is inconsistent. For the moment, I will leave this because we use it in the Install guide. I opened an issue in our Supplementary Style Guide to use "RPM-OSTree" and can update if needed.